src/Consolonia.Core/Infrastructure/DefaultNetConsole.cs

```diff
@@ -41,29 +46,20 @@ public class DefaultNetConsole : InputLessDefaultNetConsole
             (ConsoleModifiers.Shift, RawInputModifiers.Shift), (ConsoleModifiers.Alt, RawInputModifiers.Alt)
         ]);
 
-        public DefaultNetConsole()
-        {
-            StartSizeCheckTimerAsync();
-            StartInputReading();
-        }
+        public override bool SupportsAltSolo => true;
```

Comment:
Console does not recognise ALT press, correct me if I'm wrong

Reply:
you are correct.